float64[] action

---
float64[] joint_pos
float64[] state
float64[] joint_angle
float64[] limit
float64[] quaterniond
float64[] curr_angle
bool      quat_inv
bool      success
bool      singularity